components/observations/ObservationsListView.tsx

```diff
@@ -161,7 +161,11 @@ export const ObservationsListView: React.FunctionComponent<ObservationsListViewP
   );
 
   if (incompleteQueryState(nacObservationsResult, nwacObservationsResult, mapResult) || !observations || !mapLayer) {
-    return <QueryState results={[nacObservationsResult, nwacObservationsResult, mapResult]} />;
+    return (
+      <Center width="100%" height="100%">
```

Comment:
Why do we need to center _this_ `QueryState` but not others? Is there a `Center` that some higher-level component expects? Are we `Center`ing all outputs from this ListView?

Reply:
Oh, I think there's no guarantee that others don't need something similar. I'm playing whack-a-mole because I don't have a great way to test these different states. When I tackle #313 I'll build something to force different outcomes more effectively.